src/adaptors/http1/http1_server.c

```diff
@@ -218,52 +217,38 @@ static qdr_http1_connection_t *_create_server_connection(qd_http_connector_t *ct
 //
 // Note that this runs on the Management Agent thread, which may be running concurrently with the
 // I/O and timer threads.
-qd_http_connector_t *qd_http1_configure_connector(qd_dispatch_t *qd, qd_http_adaptor_config_t *config, qd_entity_t *entity)
+qd_http_connector_t *qd_http1_configure_connector(qd_http_connector_t *c, qd_dispatch_t *qd, qd_entity_t *entity)
```

Comment:
Same here. Instead of `qd_http_connector_t *c` can we have `qd_http_connector_t *connector` ?

Reply:
Yup